app/src/main/res/values/themes.xml

```diff
@@ -192,6 +192,15 @@
         <item name="android:colorForeground">?attr/settingsSwitchBackgroundColor</item>
     </style>
 
+    <style name="TextInputLayoutTheme" parent="Widget.Design.TextInputLayout">
+        <item name="colorControlActivated">@color/cornflowerBlue</item>
```

Comment:
should this be `colorAccent`?

Reply:
I want to be consistent and other themes are also referring to the specific color instead of colorAccent. `SettingsSwitchTheme` for example. Should this be revisited at some point it could be changed, but for consistency we should follow the same pattern @cmonfortep